src/components/App.js

```diff
@@ -18,12 +16,7 @@ class App extends Component {
   }
 
   componentDidMount() {
-    // function to fire fetch in apiCalls
-    // that apiCalls function would fetch, then first function in app
-    // that app function would setState
-
-    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
-      .then(results => results.json())
+    fetchAllMovies()
```

Comment:
Do you use all of the data from the fetch? If not, clean up that data to ONLY what you need before setting it to state.